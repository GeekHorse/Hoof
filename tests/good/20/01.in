new right one done
cancel
new down two done
new in three done
new in four done
new down five done
root
new right six done
value
quit
