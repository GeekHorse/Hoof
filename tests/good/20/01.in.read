hello 
new 
ok 
navigate 
new 
ok 
new 
ok 
new 
ok 
new 
ok 
ok 
new 
ok 
ok   one six 
goodbye 
