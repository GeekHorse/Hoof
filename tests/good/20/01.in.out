new right one six done
new down two done
new in three done
new in four done
new down five done
out
out
